packages/graphql/lib/services/resolvers-explorer.service.ts

```diff
@@ -279,12 +279,21 @@ export class ResolversExplorerService extends BaseExplorerService {
     };
   }
 
+  mapCtors(wrapper: InstanceWrapper): Function[] {
```

Comment:
Can we:
- move `mapCtors` below the `getAllCtors` method
- rename it to `mapToCtor`
- make it private
?